serialization.md

```diff
@@ -472,4 +472,42 @@ The resulting Tar archive for `f60c56784b83` has the following entries:
 /etc/.wh.my-app-config
 ```
 
+In addition to expressing that a single entry should be removed from a lower layer, layers may remove all of the children using an opaque whiteout entry.
```

Comment:
so, perhaps this might more clearly say something like: if the entry is a directory ...  
What do you think? Because the use of the word 'children' only applies if that is the case.
Also, I'm curious (and need to test this), but what would happen if the entry is a symlink to a directory? does this imply that only the symlink be removed?